dynamic.js

```diff
@@ -281,14 +282,17 @@ module.exports = function (fastify, opts, next) {
         const rawJsonSchema = fastifyResponseJson[key]
         const resolved = ref.resolve(rawJsonSchema)
 
+        /* istanbul ignore next */
```

Comment:
The following `if-else` is ignored as a `type` is somehow inserted under the hood. Therefore, we'll never fall into an `else` as far as I understood

Reply:
Let's drop the if.